mriqc/bin/mriqc_run.py

```diff
@@ -201,8 +215,11 @@ def main():
     log_dir = op.join(settings['output_dir'], 'logs')
 
     analysis_levels = opts.analysis_level
-    if opts.participant_label is None:
+
+    # In case direct file path is given (instead of bids dir) we do not want group level
+    if (opts.participant_label is None) & opts.T1w_file is False & opts.T2w_file is False & opts.bold_file is False :
```

Comment:
Yes this is the way it was implemented. But if no participant_label is given AND the --T1w_file flag is given, this means only a single file is processed and we do not want the group analysis (only one subject)